packages/lib-sourcify/test/verification.spec.ts

```diff
@@ -341,7 +341,7 @@ describe('lib-sourcify tests', () => {
 
       expectMatch(match, 'perfect', childAddress);
     });
-    it('should verify a contract passing files with wrong newlines', async () => {
+    it('should verify a contract with LF eol sources passing sources with CRLF eol', async () => {
```

Comment:
```suggestion
    it('should verify a contract which is originally compiled and deployed with Unix style End Of Line (EOL) source code, but being verified with Windows style (CRLF) EOL source code', async () => {
```

It's still a little vague to me. Can you write it like this? Sorry to be too intrusive, but let's lean on being more explicit and clear.